docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>I believe that this wording can be potentially controversial. What if this is the approach that the reader has come up with in the first place? Does this mean that they are now naive? In general, this is another example of why I think that coming up with strawmen and then defeating them to promote a certain narrative may be not the most effective way to communicate.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>